src/lib/transit/station/spatial-index.ts

```
@@ -0,0 +1,146 @@
+          feature.properties &&
+          typeof feature.properties.index === 'number' &&
+          feature.properties.index >= 0 &&
+          feature.properties.index < this.items.length
+      )
+      .map((feature) => this.items[feature.properties!.index]);
+  }
+
+  findNearest(
+    center: Coordinates,
+    k = 1,
+    maxRadiusMeters = Infinity
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)